### ISSUE TITLE
Wrong order in notification bubbles

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#76]
```
What steps will reproduce the problem?
The regular update of the timeline with several new tweets.

What is the expected output? What do you see instead?
The Ubuntu notification bubbles show the newer updates before the older
ones, so consecutive updates loose their original sense and are difficult
to understand. I think older updates should be first.

What version of the product are you using? On what operating system?
pino 0.1.4 on Ubuntu 9.10

Please provide any additional information below.

```

Original issue reported on code.google.com by `picom...@gmail.com` on 4 Feb 2010 at 11:20

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 5 Feb 2010 at 11:13
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 3845d46616f3 revision.
```

Original comment by `tro...@gmail.com` on 5 Feb 2010 at 1:00
- Changed state: **Fixed**
